src/Main.java

```diff
@@ -1,61 +1,144 @@
-        System.out.println("Задача 6");
-        var MassOfOneBoxer = 78.2;
-        var MassOfTheSecondBoxer = 82.7;
-        var TotalMassOfTwoFighters = MassOfOneBoxer + MassOfTheSecondBoxer;
-        System.out.println("Общая масса двух бойцов " + TotalMassOfTwoFighters + " кг");
-        var TheDifferenceBetweenTheMassesOfFighters = MassOfTheSecondBoxer - MassOfOneBoxer;
-        System.out.println("Разницу между массами бойцов " + TheDifferenceBetweenTheMassesOfFighters + " кг");
-        System.out.println("Задача 7");
-        var RemainderFromSeparation = MassOfTheSecondBoxer % MassOfOneBoxer;
-        System.out.println("Остаток от деления " + RemainderFromSeparation);
-        System.out.println("Задача 8");
-        var on = 640;
-        var two = 8;
-        var three = on / two;
-        System.out.println("1) Всего работников в компании- " + three + " человек");
-        var four = 94;
-        var sixth = three + four;
-        var fifth = two * sixth;
-        System.out.println("2) Если в компании работает " + sixth + " человек, то всего " + fifth + " часов работы может быть поделено между сотрудниками ");
+        System.out.println();
+        System.out.println("Задание 1");
+        int apple = 1000;
+        byte orange = 100;
+        short pear = 30000;
+        long potato = 1000L;
+        float cucumbers = 3.6f;
+        double plum = 1.3;
+        System.out.println();
+        System.out.println("Задание 2");
+        double a = 27.12;
+        long b = 987678965549L;
+        float c = 2.786f;
+        short d = 569;
+        short e = -159;
+        short n = 27897;
+        byte k = 67;
+        System.out.println();
+        System.out.println("Задание 3");
+        byte LyudmilaPavlovna = 23;
+        byte AnnaSergeevna = 27;
+        byte EkaterinaAndreevna = 30;
+        short totalSheetsOfPaper = 480;
+        short totalStudents = (short) (LyudmilaPavlovna + AnnaSergeevna + EkaterinaAndreevna);
+        byte sheetsForEachStudent = (byte) (totalSheetsOfPaper / totalStudents);
+        System.out.println("На каждого ученика рассчитано " + sheetsForEachStudent + " листов бумаги");
+
+        System.out.println();
+        System.out.println("Задание 4");
+        byte MachinePerformanceIn2Minutes = 16;
+        byte MachinePerformanceIn1Minutes = (byte) 16 / 2;
+        byte TwentyMinutes = 20;
+        short MachinePerformanceIn20Minutes = (short) (MachinePerformanceIn1Minutes * TwentyMinutes);
+        System.out.println("За " + TwentyMinutes + " машина произвела " + MachinePerformanceIn20Minutes + " штук");
+
+        byte hoursInADday = 24;
+        byte howManyMinutesInOneHour = 60;
+        short perDayMinutes = (short) (hoursInADday * howManyMinutesInOneHour);
+        short MachinePerformanceInPerDay = (short) (MachinePerformanceIn1Minutes * perDayMinutes);
+        System.out.println("В сутки" + " машина произвела " + MachinePerformanceInPerDay + " штук");
+
+        byte NumberOfDays = 3;
```

Comment:
```suggestion
        byte numberOfDays = 3;
```
названия переменных надо писать с маленькой буквы